angular/base/package.json

```diff
@@ -13,46 +13,46 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^7.2.2",
```

Comment:
These appear unalphabetized, which may result in unnecessary noise in version control when people `npm install`